# DP: Skip Go testcase on AArch64 which hangs on the buildds.

Index: b/gcc/testsuite/go.test/go-test.exp
===================================================================
--- a/gcc/testsuite/go.test/go-test.exp
+++ b/gcc/testsuite/go.test/go-test.exp
@@ -404,6 +404,14 @@ proc go-gc-tests { } {
 	    continue
 	}
 
+	# Hangs on the buildds
+	if { [istarget "aarch64*-*-*"] } {
+	    if { [string match "*go.test/test/chan/select5.go" $test] } {
+		untested $test
+		continue
+	    }
+	}
+
 	if { [file tail $test] == "init1.go" } {
 	    # This tests whether GC runs during init, which for gccgo
 	    # it currently does not.
